docs/core_docs/docs/deno.json

```diff
@@ -1,21 +1,18 @@
 {
   "imports": {
-    "langchain": "npm:/file:../../../langchain",
-    "langchain/": "npm:/file:../../../langchain/",
-    "@langchain/core": "npm:/file:../../../langchain-core",
-    "@langchain/core/": "npm:/file:../../../langchain-core/",
-    "@langchain/community": "npm:/file:../../../libs/langchain-community",
-    "@langchain/community/": "npm:/file:../../../libs/langchain-community/",
-    "@langchain/openai": "npm:/file:../../../libs/langchain-openai",
-    "@langchain/openai/": "npm:/file:../../../libs/langchain-openai/",
-    "@langchain/mistralai": "npm:/file:../../../libs/langchain-mistralai",
-    "@langchain/mistralai/": "npm:/file:../../../libs/langchain-mistralai/",
-    "@langchain/anthropic": "npm:/file:../../../libs/langchain-anthropic",
-    "@langchain/anthropic/": "npm:/file:../../../libs/langchain-anthropic/",
-    "@langchain/cohere": "npm:/file:../../../libs/langchain-cohere",
-    "@langchain/cohere/": "npm:/file:../../../libs/langchain-cohere/",
-    "@langchain/google-genai": "npm:/file:../../../libs/langchain-google-genai",
-    "@langchain/google-genai/": "npm:/file:../../../libs/langchain-google-genai/",
+    "langchain/": "npm:langchain/",
+    "@langchain/core/": "npm:langchain/core/",
```

Comment:
This is wrong but let's also really try to figure out how to do this pulling in local code

Reply:
whoops, fixed. and agreed.